base/src/main/java/com/smartdevicelink/managers/screen/choiceset/PreloadChoicesCompletionListener.java

```diff
@@ -0,0 +1,13 @@
+package com.smartdevicelink.managers.screen.choiceset;
+
+import java.util.HashSet;
+
+public interface PreloadChoicesCompletionListener {
```

Comment:
Can you make this internal to BaseChoiceSetManager like what you did on this PR: https://github.com/smartdevicelink/sdl_java_suite/pull/1716/files, also just create one interface to use, as this does the same thing as the `DeleteChoicesCompletingListener` so something like: 

```
    interface ChoiceCompletionListener {
        void onComplete(boolean success, HashSet<ChoiceCell> failedChoiceCells);
    }
```